.github/workflows/check.yml

```diff
@@ -9,13 +9,17 @@ on:
 
 jobs:
   check:
-
     runs-on: ubuntu-latest
+
     strategy:
       fail-fast: false
       matrix:
         php-version: [7.2, 7.3, 7.4, 8.0, 8.1]
         protocol: [ 'json', 'msgpack' ]
+        ignorePlatformReq: [ '' ]
+        include:
+          - php-version: 8.2
+            ignorePlatformReq: '--ignore-platform-reqs'
```

Comment:
better would be to use `--ignore-platform-req=php+` to ignore only the upper bound of the PHP requirement instead of ignoring all platform requirements

Reply:
Done!